inc/classes/wp-rocket-uninstall.php

```diff
@@ -0,0 +1,195 @@
+<?php
```

Comment:
Move this into `Engine` as `classes` will eventually be gone.

Reply:
Fixed https://github.com/wp-media/wp-rocket/pull/2476/commits/f5180d716d294f8d4122c6486ae28621546262f5